src/compas/artists/meshartist.py

```diff
@@ -79,16 +137,26 @@ class MeshArtist(Artist):
 
     """
 
-    default_color = (0.0, 0.0, 0.0)
-    default_vertexcolor = (1.0, 1.0, 1.0)
-    default_edgecolor = (0.0, 0.0, 0.0)
-    default_facecolor = (0.9, 0.9, 0.9)
+    color = Color.from_hex('#0092D2').lightened(50)
+
+    default_vertexcolor = Color.from_hex('#0092D2')
+    default_edgecolor = Color.from_hex('#0092D2')
+    default_facecolor = Color.from_hex('#0092D2').lightened(50)
+
+    vertex_color = ColorDict()
+    edge_color = ColorDict()
+    face_color = ColorDict()
```

Comment:
just nitpicking, should it be `vertexcolor` here since it was `default_vertexcolor` above?

Reply:
nevermind...it might be too much a breaking change...

Reply:
well, i have been using the convention that the underscore form indicates a map between first and second part. so `vertex_color` is a map between `vertex` and `color`. but perhaps i was overthinking this :)